src/FAQ/FaqData.js

```diff
@@ -1,40 +1,55 @@
-     link: <a className="color-navigation font-weight-bold"target="blank"href="https://levelup.charite.de/app-demo">https://levelup.charite.de/app-demo</a>
-},
-{
-    question: 'Kann ich LevelUp vom Handy aus nutzen?',
-    answer: 'Ja, LevelUp lässt sich ganz bequem  vom Handy aus nutzen. Einfach im Handy-Browser die Adresse aufrufen. LevelUp ist keine App, sondern wird im Browser dargestellt. Somit wird kein Speicherplatz für die App auf Deinem Handy benötigt.'
-},
-{
-    question: 'Meine letzten Prüfungen werden nicht dargestellt, warum nicht?',
-    answer: 'Die Ergebnisse der Prüfungen werden erst nach einigen Wochen in LevelUp dargestellt. Teilweise kann es leider auch einige Monate dauern, wie beim PTM. Nicht davon betroffen sind die Daten aus HIS wie Anwesenheiten und Meilensteine wie Pflegepraktikum etc. Diese werden spätestens 2 Tage nach der Darstellung in HIS auch in LevelUp angezeigt.'
-},
-{
-    question: 'Eines meiner Prüfungsergebnisse stimmt nicht mit den Daten im HIS überein. Woran liegt das?',
-    answer: 'Möglicherweise handelt es sich um einen Importfehler. Bitte kontaktiere in diesem Fall den Support unter',
-    email :<a className="color-navigation font-weight-bold"href="mailto:levelup@charite.de">levelup@charite.de</a>
-},
-{
-    question: 'Ich kann mich mit meinen Charité-Logindaten im LevelUp nicht anmelden/registrieren.',
-    answer:'Bist Du aktuell im MSM 2.0 immatrikuliert? Wenn ja,  kontaktiere den Support unter',
-    email: <a className="color-navigation font-weight-bold"href="mailto:levelup@charite.de">levelup@charite.de</a>
-}
+    {
+        question: 'Was ist LevelUp und wofür brauche ich das Tool?',
+        answer: 'LevelUp ist ein neues Tool für alle Studierenden im MSM 2.0 an der Charité. Du siehst hier sämtliche Prüfungsergebnisse und deinen Lernfortschritt im Studium. Es zeigt deine MC-Prüfungsergebnisse bis ins Detail, inkl. Fragen, Antworten und Lösungen. Die praktischen ärztlichen Tätigkeiten (EPAs), die im Studium gelernt werden, werden hier dokumentiert und bewertet, sowohl durch dich selbst, als auch deine Dozierenden. Du kannst außerdem deinen Wissenszuwachs anhand des PTM-Features verfolgen. Dein Wissen wird auch nach Fachgebiet gerankt und du siehst wo aktuell deine Stärken sind, und wo du noch üben musst.'
+    },
+    {
+        question: 'Wie nutze ich LevelUp?',
+        AnswerWithLinks: ['Hier', ' geht es zum Tutorial für das Tool, und ', 'Hier', ' geht es zu einem kurzen Clickvideo, in dem gezeigt wird, wie man auf der Seite navigiert.'],
+        tutorialLink: 'https://levelup.charite.de/app/tutorial/src/index.html',
+        clickvideoLink: 'https://levelup.charite.de/videos/Klickvideo.mp4'
```

Comment:
Sorry, aber das ist nicht so eine gute Architektur. Schreibe morgen noch mehr dazu. LG!

Reply:
Ich habe keine andere Lösung für diese Stelle gefunden, um den Tages rauszuschmeißen .
Wenn du eine Idee hast, gerne :)

Reply:
Mit "nicht so gute Architektur" meine ich folgendes: 

Der Code hat schon eine bestimmte Struktur. Wenn etwas vorkommt, was in diese Struktur nicht reinpasst, sollte die Struktur geändert werden. 
Du hast die bestehende Code-Struktur nicht geändert, sondern in deiner Änderung ist ein  Spezialfall außerhalb der Code-Struktur hinzugefügt. `tutorialLink` und `clickvideoLink` gibt es nur in dieser Frage. Alle anderen Fragen haben falls nötig das Attribut `email` oder `link`, und es wird wahrscheinlich auch nie wieder eine Frage geben, die deine neuen Attribute braucht. Darum Spezialfall, und Spezialfälle wollen wir vermeiden.
Außerdem hat auch die für den Nutzer sichtbare UI-Struktur einen Spezialfall dazubekommen. Bei den Mail-Links und beim Demo-Link ist die Adresse für den Nutzer sichtbar, und sie steht am Ende der Antwort. In deiner neuen Version steht nur "Hier", und die zwei "Hier" sind im Antwort-Text versteckt und dadurch viel weniger sichtbar. 

Probiere bitte bei deinen Tickets die Strukturen im Code und in der UI so wenig wie möglich zu verändern. 

Diese Frage sollte diese Struktur haben: 

```
{
        question: 'Wie nutze ich LevelUp?',
        answer: 'Hier sind Links zum Tutorial für das Tool, und zu einem kurzen Clickvideo, in dem gezeigt wird, wie man auf der Seite navigiert:',
        links: [
		'https://levelup.charite.de/app/tutorial/src/index.html',
        	'https://levelup.charite.de/videos/Klickvideo.mp4'
		]
}
```

Damit ist bleibt die UI-Sturktur gleich, und du verwendest das alte Attribut `answer` anstatt ein neues einzuführen. 
Das neue Attribut `links` ersetzt das alte `link`. Bei der Frage "Kann ich LevelUp ansehen, wenn ich nicht studiere?" gleichst du die Struktur an, aus `link` wird `links` und den Wert tust du in ein array. 
Das alte Attribut `email` bleibt wie es ist. 

`AnswerWithLinks` und `Link` wird dann unnötig, siehe mein Kommentar oben.

Reply:
Hi Nika, danke für die Review aber ich stimme dir nicht völlig zu
1-	Ich habe die Struktur nicht ausgedacht sondern es steht im Tickt so, dass „ Hir“ verlinkt wird.
https://llp-op.charite.de/projects/levelup/wiki/faqs-erweiterung
Als Programmierer habe ich keinen Anspruch, den Text einfach zu ändern, nur um einen Variablenname ändern zu vermeiden. sondern nur umsetzen, was im Tickt steht. Wenn ich etwas nicht umsetzten könnte oder ich gegen den Text was habe, da könnte ich nach Änderung bitten.
Du hast das Problem gelöst, indem du einfach den Text geändert!
Vielleicht hast du ja Recht mit UI Struktur, dann würde ich dich bitten , deinen Vorschlag im Tickt zu schreiben. Wenn die Änderung von Hanna (feature owner) genehmigt wird und das Tickt geänderte wird , dann können wir den Text ändern. Vielleicht Hanna hat eine andre Meinung 

2-	Bitte zwischen den alten und neuen Code unterscheiden, du hast alten Code kommentieret 
3-	Mit AnswerWithLinks  und  tutorialLink hast du ja Recht 
LG

Reply:
Hi Toma, 
1 - stimmt - es steht im Ticket drin. Ich hatte es nur nicht gesehen. Irgendwie war mir gar nicht klar, dass es diese Wiki-Seite "FAQs Erweiterung" gibt, denn ich hatte die Kommentare am Ticket nicht gelesen. 
In Zukunft beschränke ich mich mal darauf, was ein Reviewer macht: nur den Code lesen und kommentieren. Als Reviewer sollte ich eigentlich gar nicht in das Ticket gucken. Das war mein ursprünglicher Fehler und dann hab ich das Ticket auch noch falsch gelesen. 
2 - ich dachte, ich habe nur an den Stellen den alten Code kommentiert, wo ich direkt darauf Bezug genommen habe... Also es ging in meinem Kommentar ja darum, sich an den alten Code zu halten. Habe ich eine Änderung kommentiert, die du schon überschrieben hattest? 
Die Frage ist jetzt nur allgemein gemeint - dieses Merge Request ist eh ok, weil das Ticket eben so ist wie es ist. 
3 - naja, falls dir noch was besseres einfällt, was trotzdem mit dem Ticket übereinstimmt, wäre es natürlich prima. 

Ich approve das hier erstmal. Wenn du noch was ändern möchtest, mach gern eine neue Merge Request auf. Wenn du nichts ändern möchtest, dann merge einfach.

Reply:
alles klar